resources/components/App.vue

```diff
@@ -119,10 +130,10 @@ module.exports = {
 		 * Watch the tab change events emitted by the <Tabs> component
 		 * to ensure that Vuex state is kept in sync
 		 *
-		 * @param {string} tab name
+		 * @param {VueComponent} tab
```

Comment:
I acknowledge this is a bit weird and am open to alternatives if we can come up with any.

Basically, the problem here is that we need for some custom logic to happen when the user changes tabs. Since the Tabs component is designed to be re-usable, we can't make any assumptions there about what kinds of content we'll be receiving.

But we do know that the Tab component is going to have slot content of some sort. When the active Tab in the Tabs component gets changed, the entire now-active tab is passed up as the payload of the `tab-change` event, which allows us to access whatever it is that we might care about.